microsoft-edge/devtools-guide-chromium/console/console-dom-interaction.md

```diff
@@ -5,290 +5,345 @@ author: MSEdgeTeam
 
-   ![The $$ shortcut returns far too much information](./console-dom-interaction-images/console-dom-too-much-link-information.png)
+   Expand a few of the returned link objects to see their properties:
 
-   To solve that problem, the `$$` shortcut has an interesting extra feature: instead of returning a pure `NodeList` like `document.querySelectorAll()`, the `$$` shortcut gives you all of the `Array` methods, including `map()`.
+   ![The list of links on the page, with each link's text and href attribute](./console-dom-interaction-images/mapped-links.png)
 
-1. Use the `map()` method of the `Array` object to reduce the information to what you need:
+1. To only list external links, use the `filter()` array method to get rid of the links that don't start with `https`:
 
    ```javascript
    $$('a').map(a => {
-      return {url: a.href, text: a.innerText}
-   })
+      return {url: a.getAttribute("href"), text: a.innerText}
+   }).filter(a => {
+      return a.url.startsWith("https");
+   });
    ```
 
-   The above code returns an `Array` of all the links, as objects with `url` and `text` properties.
+   ![The list of external links on the page](./console-dom-interaction-images/external-links.png)
+
+
+###### Display DOM elements in a table
+
+In the previous example, you displayed a list of elements from the page in the **Console**. The elements are displayed as a JavaScript array. To make it easier to use the list of elements in the **Console**, you can display them in a sortable table instead, by using the `console.table()` utility function:
 
-   ![Use map on $$ to filter information down to the bare minimum](./console-dom-interaction-images/console-dom-filter-link-data.png)
+1. Go to the webpage that you want to inspect. For example, open the [DevTools Console tool DOM interactions demo page](https://microsoftedge.github.io/Demos/devtools-console-dom-interactions/) in a new tab or window.
 
-   You aren't done yet; several links are internal links to the webpage or have empty text.
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The **Console** tool opens in DevTools, next to the webpage.
 
-1. Use the `filter` method to get rid of the internal links:
+1. Paste the following code into the **Console**, and then press **Enter**:
 
    ```javascript
-   $$('a').map(a => {
-      return {text: a.innerText, url: a.href}
-   }).filter(a => {
-      return a.text !== '' && !a.url.match('learn.microsoft.com')
-   })
+   console.table($$('img'), ['src','alt'])
    ```
 
-   ![Get the links that aren't empty and are external](./console-dom-interaction-images/console-dom-filter-out-empty-links.png)
+   The list of `<img>` elements on the webpage is displayed in the **Console**, formatted as a table, with each image's `src` and `alt` attributes as columns:
+
+   ![The console tool, showing the console.table() function and a table listing all images](./console-dom-interaction-images/images-in-table.png)
```

Comment:
what animal is "A socke"??  searched web, didn't find.
rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/console-dom-interaction?branch=pr-en-us-2881#display-dom-elements-in-a-table
I guess it means this dog: 
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/eb1c3789-0052-4209-885b-853737e20fd1)
Seems like the demo needs revision b/c "A socke" in a list of animals is confusing.